### PR TITLE
Кириллов К.И. 381806-2. ЛР3

### PR DESCRIPTION
**Pull request by kustuk1306**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [x] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by kustuk1306** [comment]
@LaQerty , @mercy131 Посмотрите, пожалуйста

**Comment by kustuk1306** [comment]
@OnlyDeniko

File: modules/general-subseq/src/GeneralSubseqApp.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+GeneralSubseqApp::GeneralSubseqApp() : message_("") {}
+
+void GeneralSubseqApp::help(const char* appname, const char* message) {
+    message_ =
+        std::string(message) +
+        "This is a check of the longest general subsequential search," +
+        "among a randomly given sequence.\n\n" +
+        "Please provide arguments in the following format:\n$"
+        + appname + "count . \n\n"
+        + "The argument is the length of the generated sequence\n\n" \
```
Comment (LaQerty) [inline modules/general-subseq/src/GeneralSubseqApp.cpp:23]: Замени + в конце строк на \

File: modules/general-subseq/src/GeneralSubseqApp.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 Kirillov Konstantin
+
+#include <stdlib.h>
+#include <iostream>
+#include <string>
+#include <ctime>
```
Comment (mercy131) [inline modules/general-subseq/src/GeneralSubseqApp.cpp:6]: Убери лишние библиотеки.

**Review by mercy131 — APPROVED** [review]

**Review by LaQerty — APPROVED** [review]

**Review by OnlyDeniko — APPROVED** [review]

**Pull request merged**